torch/jit/_script.py

```diff
@@ -478,13 +478,13 @@ def code_with_constants(self):
             r = self.forward.code_with_constants
             return (r[0], ConstMap(r[1]))
 
-        def save(self, f, _extra_files={}):
+        def save(self, f, **kwargs):
```

Comment:
OOC, why is this needed?

Reply:
The lint error was because I was passing a mutable object (i.e. `{}`) as a default argument. Alternative solution would be to have `None` as the default with
```python
if _extra_files is None:
    _extra_files = {}
```
inside the function. But before I made any changes the function was already using kwargs so I figured I'd just switch back to that.